File: test/fixtures/csharp/put_xput.cs
using System.Net.Http;
using System.Net.Http.Headers;

HttpClient client = new HttpClient();

HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Put, "http://localhost:28139/twitter/_mapping/user?pretty");
request.Content = new StringContent("{\"properties\": {\"email\": {\"type\": \"keyword\"}}}");
request.Content.Headers.ContentType = new MediaTypeHeaderValue("application/json");

HttpResponseMessage response = await client.SendAsync(request);
response.EnsureSuccessStatusCode();
string responseBody = await response.Content.ReadAsStringAsync();

File: test/fixtures/javascript/put_xput.js
fetch('http://localhost:28139/twitter/_mapping/user?pretty', {
  method: 'PUT',
  headers: {
    'Content-Type': 'application/json'
  },
  // body: '{"properties": {"email": {"type": "keyword"}}}',
  body: JSON.stringify({
    'properties': {
      'email': {
        'type': 'keyword'
      }
    }
  })
});
